### PR TITLE
Login, Register API path code fix

### PR DESCRIPTION
**Pull request by firen777**
To properly connect to the host, change 'host' constant in src/environments/apipath.ts

### CODE REVIEW

File: peer-tutor-ui/angular/src/environments/environment.ts [hunk 1]
```diff
@@ -3,11 +3,34 @@
...
 
 //TODO: change the API URL to actual backend!
+/**
+ * apiUrl need to be configured for every new run!
+ */
 export const environment = {
   production: false,
-  apiUrl: 'http://localhost:4200'
+  // apiUrl: 'http://localhost:4200',
+  apiUrl: 'http://192.168.0.100:5000',
```
Comment (bharathkkb) [inline peer-tutor-ui/angular/src/environments/environment.ts:12]: So is localhost not working? This ip will break for everybody else other than your machine. We can merge with this now, but best to put a  comment or something here

**Comment by firen777** [comment]
Test Comment

File: peer-tutor-ui/angular/src/environments/environment.ts (hunk 1 above)
Comment (bharathkkb) [reply peer-tutor-ui/angular/src/environments/environment.ts:12]: check if u can see now @firen777

Comment (firen777) [reply peer-tutor-ui/angular/src/environments/environment.ts:12]: actually im not using environment.apiUrl (will delete it once all the fake backend request got removed). It is literally useless.

The current path using is in src/environments/apipath

Comment (firen777) [reply peer-tutor-ui/angular/src/environments/environment.ts:12]: http://firen777-networklab.ddns.net:5000/

This is the one I'm hosting on my desktop

In production, should the backend also host on the same machine or on a different server?

(Also, localhost work for me. I was just playing with it and forgot to change it back)

Comment (bharathkkb) [reply peer-tutor-ui/angular/src/environments/environment.ts:12]: So for production we would need to pass variable by commandline kinda like this 
https://medium.com/@aziz.marwan/angular-cli-environment-variables-75f926c52798
so for production we would do ng build --environment=prd
for local we would do ng build --environment=dev
for now its fine, I just wanted to give you a heads up

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**